### PR TITLE
docs: improving eventing testkit configuration documentation

### PR DESCRIPTION
**Pull request by aludwiko**
References #1845

### CODE REVIEW

File: docs/src/modules/java/pages/actions-publishing-subscribing.adoc [hunk 1]
```diff
@@ -214,6 +214,17 @@ include::example$java-spring-eventsourced-counter/src/it/java/com/example/Counte
 
 TIP: In the example above we take advantage of the TestKit to serialize / deserialize the messages and pass all the required metadata automatically for us. However, the API also offers the possibility to read and write raw bytes, construct your metadata or read multiple messages at once.
 
+==== Configuration
+
+Before running your test, make sure to configure the TestKit correctly.
+
+[source, java]
+----
+include::example$java-spring-eventsourced-counter/src/it/java/com/example/TestkitConfig.java[tags=class;eventing-config]
```
Comment (efgpinto) [inline docs/src/modules/java/pages/actions-publishing-subscribing.adoc:223]:
```suggestion
include::example$java-spring-eventsourced-counter/src/it/java/com/example/TestKitConfig.java[tags=class;eventing-config]
```

**Review by efgpinto — APPROVED** [review]
LGTM with small typo fix to please CI

**Review by octonato — APPROVED** [review]
LGTM

File: samples/java-spring-eventsourced-counter/src/it/java/com/example/TestKitConfig.java
Comment (octonato) [inline samples/java-spring-eventsourced-counter/src/it/java/com/example/TestKitConfig.java:1]: this file was renamed to contain an uppercased `Kit`, but there are some references still pointing to `kit`

**Pull request merged**